### PR TITLE
Lab9

### PR DESCRIPTION
**Pull request by omostovyijr**

### CODE REVIEW

File: lecture_notes/09 functions/2-scope.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+let cities: [String] = ["Athens", "Roma", "London", "Beijing", "Kiev", "Riga"]
+let citiesLengths: [Int] = cities.map {$0.count}
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/2-scope.swift:4]: You have t choices
- Improve your understanding of swift syntax
- Rewrite this line to well known constructions like for

File: lecture_notes/09 functions/3-default.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+max = lambda a = 0, b = 0: a if a > b else b
+
+print(max(10,20), max(10), max(-20))
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/3-default.py:3]: \n

File: lecture_notes/09 functions/3-default.swift [hunk 3]
```diff
@@ -0,0 +1,10 @@
+func max(a: Int = 0, b: Int = 0) -> Int{
+    if (a>b) {
+        return a
+    }
+    else {
+        return b
+    }
+}
```
Comment (tshemsedinov) [inline lecture_notes/09 functions/3-default.swift:8]:
```suggestion
    if (a > b) {
        return a
    }
    return b
}
```

**Review by tshemsedinov — APPROVED** [review]

File: lecture_notes/09 functions/2-scope.swift (hunk 1 above)
Comment (omostovyijr) [reply lecture_notes/09 functions/2-scope.swift:4]:
```swift
var citiesLengths: [Int] = []

func cityLength() {
    for city in cities {
        let cityLength = city.count
        citiesLengths.append(cityLength)
    }

}
```

File: lecture_notes/09 functions/2-scope.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    var cityUpperTwo: [String] = []
+    for city in citiesTwo {
+        cityUpperTwo.append(city.uppercased())
+    }
+    print(citiesTwo)
+    print(cityUpperTwo)
+}
+
+f1()
+
```
Comment (omostovyijr) [inline lecture_notes/09 functions/2-scope.swift:34]:
```swift
cityLength()
```

**Pull request closed without merging**